Kronos-Shifts-Connector/Deployment/azuredeploy.json

```diff
@@ -587,18 +585,53 @@
             ]
         },
         {
-            "name": "[variables('keyVaultName')]",
             "type": "Microsoft.KeyVault/vaults",
-            "apiVersion": "2016-10-01",
+            "apiVersion": "2018-02-14",
+            "name": "[variables('keyVaultName')]",
             "location": "[parameters('location')]",
+            "dependsOn": [
+                "[concat('Microsoft.Web/sites/', variables('apiAppName'))]",
+                "[concat('Microsoft.Web/sites/', variables('configAppName'))]"
+            ],
+            "tags": {
+                "displayName": "KeyVault"
+            },
             "properties": {
-                "sku": {
-                    "family": "A",
-                    "name": "Standard"
-                },
+                "enabledForDeployment": "true",
+                "enabledForTemplateDeployment": "true",
+                "enabledForDiskEncryption": "true",
                 "tenantId": "[subscription().tenantId]",
-                "enabledForTemplateDeployment": true,
                 "accessPolicies": [
+                    {
+                        "objectId": "[reference(concat('Microsoft.Web/sites/', variables('apiAppName')), '2018-02-01', 'Full').identity.principalId]",
+                        "tenantId": "[subscription().tenantId]",
+                        "permissions": {
+                            "secrets": [
```

Comment:
Lock these down to only the necessary permissions. Does the API really need full access, and to everything?
Same with the config app.

Reply:
@aosolis - ack'd, and the API does not need access to everything, and as you said, the same goes for the config app. I have made the necessary changes to lock down necessary permissions

Reply:
Where are these changes.

Reply:
The changes have been committed on this branch, but not synced. I'll be pushing all changes once I have the README.md changes done as well

Reply:
@coderkrishna Why are you granting these apps permissions to keys and certificates? Does this KV even contain those?

Reply:
@coderkrishna And why are you granting them all the permissions to do all operations on secrets? Please justify the inclusion of each one.